app/store/configureStore.dev.js

```diff
@@ -3,8 +3,19 @@ import reduxImmutableStateInvariant from 'redux-immutable-state-invariant';
 import thunkMiddleware from 'redux-thunk';
 import rootReducer from '../reducers';
 
+
 // Download redux dev tools here: https://github.com/zalmoxisus/redux-devtools-extension
 
+// const store = createStore(
```

Comment:
I want to merge this into master, so if we could either uncomment this, or remove it (and then make an additional issue to track it), that would be great.